### ISSUE TITLE
space-index-pages-free-plot doesn't work

### ISSUE THREAD

**Issue by kysnm** [jeremycole/innodb_ruby#31]
I'm trying to space-index-pages-free-plot, but error occured like below.
```
➜  innodb_ruby git:(master) ✗ ruby -r rubygems -I lib bin/innodb_space -f spec/data/hello_world.ibd space-index-pages-free-plot
bin/innodb_space:1817:in `basename': no implicit conversion of Array into String (TypeError)
	from bin/innodb_space:1817:in `<main>'
```

I think this commit is affecting
https://github.com/jeremycole/innodb_ruby/commit/d8c5a95ae9f6fede5aac53dc16202941e6ff881f

Would you please fix it?

**Comment by jeremycole** [comment]
Howdy kysnm! Thanks for the report! I have fixed this in bdc5e38 – could you check it out?

**Comment by kysnm** [comment]
I checked it. Thanks for fix it. It's run correctly!
Sorry for late reply.
